models/gc_spider_baseline/meta.json

```diff
@@ -0,0 +1,219 @@
+{
+  "id": "aed5a82e-2188-4d50-83f9-45bce09f9daa",
+  "name": "gc_spider_baseline",
+  "title": "SPIDER Baseline algorithm: Iterative Instance Segmentation of the spine",
+  "summary": {
+    "description": "This model segments three anatomical structures in lumbar spine MRI: vertebrae, intervertebral discs (IVDs), and spinal canal. Each vertebrae and IVD is labeled according to standard naming convention.",
+    "inputs": [
+      {
+        "label": "Sagittal Lumbar Spine T1 T2 MRI",
+        "description": "Sagittal T1 or T2 weighted MRI of the lumbar spine. It is important to provide the input image in sagittal orientation for proper functioning of the model.",
```

Comment:
> It is important to provide the input image in sagittal orientation for proper functioning of the model
This should also go into the intended use section.

Reply:
It was mentioned in the Intended use section, but I've made it more explicit.